IUBLibTech/ngao
# Ar 18 add import ead

# Summary 

Add ability to index individual EAD files from the Admin page

# Related Issue

https://bugs.dlib.indiana.edu/browse/AR-18

# Expected Behavior

As a logged in admin user, you can view a page that allows you to import/index individual EAD files for import/indexing
 
# Screenshots / Video

![image](https://user-images.githubusercontent.com/18175797/79934103-97393380-8406-11ea-914e-e9696a111658.png)

# Dependencies

Relies on Delayed Job working in staging

# Testing / Reproduction instructions

1. Login as an admin or manager
2. You will see buttons next to each EAD file, grouped by repository
3. Click button to index

You can view jobs in the delayed job queue at https://staging-ngao-web.notch8.cloud/delayed_job/overview

# Deployment

This will be deployed to Notch8 staging site after approved and merged.

# Other Information

This is the second PR for ticket AR-18.
